### PR TITLE
support vector line layer

### PR DESCRIPTION
**Pull request by anzhihun**

### CODE REVIEW

File: src/layer/lineLayer.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import { Observable } from '../util/observable';
+import { Line as RenderableLine } from '../renderable/line';
+import { LineLayerRender } from '../render/lineLayerRender';
+import { LineMesh } from '../mesh/lineMesh';
+
+export class LineLayer extends Observable {
```
Comment (zhangwenjin) [inline src/layer/lineLayer.js:6]: vectorLayer?

File: src/source/sourceLayer.js [hunk 2]
```diff
@@ -1,12 +1,25 @@
...
-        });
+        if (layerConfig.type === Const.LayerType.RASTER_TILE) {
+            return new RasterTileLayer({
+                source: new TileSource(layerConfig.url),
+                view: { zoom: 3 },
+                context
+            });
+        } else if (layerConfig.type === Const.LayerType.LINE) {
+            return new LineLayer({
+                source: new VectorSource(layerConfig.source),
```
Comment (zhangwenjin) [inline src/source/sourceLayer.js:17]: VectorSource whether equivalent VectorLayer?understand easy

Comment (anzhihun) [inline src/source/sourceLayer.js:17]: Layer containers source, and define display styles

File: src/layer/lineLayer.js (hunk 1 above)
Comment (anzhihun) [inline src/layer/lineLayer.js:6]: LineLayer is one of the VectorLayer.  We will add TextLayer, IconLayer, CircleLayer and others.

**Pull request merged**